apps/crawler/lib/crawler/client_behaviour.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Crawler.Client do
+  @moduledoc """
+  beaviour to implement search providers for `Crawler.TaskManager`
```

Comment:
beaviour -> behaviour (or behavior)

actually, (and this is a question for another pr) we should decide on british vs american spellings at some point. for now it doesn't matter though

Reply:
also, IMO implement -> describe

i say this because a behavior lays out how an implementation should be written rather than actually implementing anything itself